components/nexus-core/src/main/java/org/sonatype/nexus/proxy/events/AbstractVetoableEvent.java

```diff
@@ -16,16 +16,11 @@
 import java.util.Collections;
 import java.util.List;
 
-import org.sonatype.nexus.events.AbstractEvent;
-
 public class AbstractVetoableEvent<T>
-    extends AbstractEvent<T>
-    implements Vetoable
 {
   private final ArrayList<Veto> vetos = new ArrayList<Veto>();
 
   public AbstractVetoableEvent(T component) {
-    super(component);
```

Comment:
Consider removing this constructor as it doesn't do anything anymore, and just means sub-classes have to keep using`super(someObject)` in their constructor.